src/WebJobs.Extensions.DurableTask/Bindings/BindingHelper.cs

```diff
@@ -130,6 +131,14 @@ private class OrchestrationClientInputData
             /// </summary>
             [JsonProperty("rpcBaseUrl")]
             public string? RpcBaseUrl { get; set; }
+
+            /// <summary>
+            /// The base URL of the Azure Functions host, used in the out-of-proc model.
+            /// This URL is sent by the client binding object to the Durable Worker extension,
+            /// allowing the extension to know the host's base URL for constructing complete URLs.
```

Comment:
nit - we can make this more concrete
```suggestion
            /// allowing the extension to know the host's base URL for constructing management URLs.
```